code/modules/mob/living/carbon/human/human.dm

```diff
@@ -726,4 +726,34 @@
 	staticOverlay.override = 1
 	staticOverlays["letter"] = staticOverlay
 
+mob/living/carbon/human/throw_impact(A)
+	..()
+
+	if(A)
+		if(istype(A, /mob/living/carbon/human)) //Humans can't tackle aliens and borgs

```

Comment:
yeah...having the leap fail because your target had a paper in his hand but work on an empty handed hulk is way too inconsistent to me.